scripts/ChatterboxGetSpeech/ChatterboxGetSpeech.gml

```diff
@@ -0,0 +1,27 @@
+/// Returns the string after the first colon in a Chatterbox's content.
+///
+/// @param chatterbox
+/// @param contentIndex
+
+function ChatterboxGetSpeech(_chatterbox, _index)
+{
+    var _str = ChatterboxGetContent(_chatterbox, _index);
+    var _speech = _str;
+    if (_str != undefined)
+    {
+        var _colon = string_pos(":", _str);
+        var _space = 0;
+        if (_colon != 0) //If there *is* a colon
+        {
+            _speech = string_delete(_str, 1, _colon); //Remove the colon and all behind it,
+        }
+        if (string_pos(" ", _speech) == 1) //If there's as the first character
```

Comment:
Chatterbox automatically strips leading whitespace from every line during compilation. All string manipulation should only happen if there's a colon.